script.js

```diff
@@ -4,4 +4,284 @@ window.addEventListener('hashchange', () => {
   } else {
     document.body.classList.remove('header--open-menu');
   }
-});
\ No newline at end of file
+});
+
+// Popup window
+
+const projectCard = [
+  {
+    id: '0',
+    image: './images/work-0.png',
+    title: 'Tonic',
+    creatorName: 'Canopy',
+    creatorPos: 'Back End Dev',
+    creatorYear: '2015',
+    descriptionShort:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    descriptionLong1:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.",
+    descriptionLong2:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s.',
+    language1: 'HTML',
+    language2: 'Ruby on rails',
+    language3: 'CSS',
+    language4: 'Javascript',
+    language5: 'Bootstrap',
```

Comment:
[OPTIONAL] Please use `languages: ["HTML", "Ruby on Rails", "CSS", "JavaScript", "Bootstrap"]` instead of `language1,2...` Please implement it like that this is optional but recommended please don't skip it because it makes your code more readable and clean :slightly_smiling_face: check below :point_down:  my suggestions :+1: 

```suggestion
    languages: ["HTML", "Ruby on Rails", "CSS", "JavaScript", "Bootstrap"]
```